jenkins/tippecanoe_create_tiles_Jenkinsfile

```diff
@@ -42,7 +42,7 @@ pipeline {
       }
       steps {
         sh '''
-          tippecanoe -zg --no-tiny-polygon-reduction --simplify-only-low-zooms --detect-shared-boundaries --simplification=5 --force --output-to-directory ${WORKSPACE}/tile_dir_simple5 ${WORKSPACE}/hrus.geojson
```

Comment:
woah, maybe that is where those crazy gaps came from, not sure how long this said 'detect-shared-boundaries' instead of 'detect-shared-borders'

Reply:
💥 Good catch.  Annoying that it doesn't error with an unknown option